Access Science.js

```diff
@@ -0,0 +1,398 @@
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	let title = attr(doc, 'meta[name="citation_title"]', 'content');
+	if (title) {
+		if (doc.querySelector('meta[name="citation_isbn"]')) {
+			let bookTitle = attr(doc, 'meta[name="citation_book_title"]', 'content');
+			if (!bookTitle || title == bookTitle) {
+				return "book";
+			}
+			else {
+				return "bookSection";
+			}
+		}
+		else if (url.includes('content/video/') || url.includes('content/video-biography')) {
+			return 'videoRecording';
+		}
+		else if (url.includes('content/article/')) {
+			return "journalArticle";
+		}
+		else if (url.includes('news') || url.includes('briefing')) {
+			return "magazineArticle";
+		}
+		else {
+			return "webpage";
+		}
+	}
+	else if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+	return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = doc.querySelectorAll('.search-middle-right a[href]');
+	for (let row of rows) {
+		let href = row.href;
+		let title = ZU.trimInternal(row.textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (items) ZU.processDocuments(Object.keys(items), scrape);
+		});
+	}
+	else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	translator.setDocument(doc);
+	translator.setHandler('itemDone', function (obj, item) {
+		// Author
+		// Some of video pages having old content which does not contain the
+		// firstname and lastname. which is binding in a single string in
+		// metadata tags, So those cases we were split and mapped accordingly
+		if (item.itemType == 'videoRecording') {
+			let authorName = attr(doc, 'meta[name="citation_author"]', 'content');
+			if (authorName) {
+				item.creators = [];
+				if (authorName.includes(',') && authorName.split(',').length > 2) {
+					authorName = authorName.split(',')[0];
+					item.creators.push(ZU.cleanAuthor(authorName, "author", false));
+				}
+			}
+		}
+
+		// Abstract contain html element
+		let abstractNote = attr(doc, 'meta[name="citation_abstract"]', 'content');
+		let isHTML = RegExp.prototype.test.bind(/(<([^>]+)>)/i);
+		if (isHTML(abstractNote)) item.abstractNote = ZU.cleanTags(abstractNote);
```

Comment:
This can just be `let htmlRE = /(<[^>]+)>/;` and `htmlRE.test(abstractNote)`. (But I'm not totally sure what this is for. Was the goal just to omit the "AccessScience is an authoritative and dynamic online resource" strings?)